datadog/resource_datadog_dashboard.go

```diff
@@ -8068,7 +8068,6 @@ func buildDatadogToplistStyle(terraformToplistStyle map[string]interface{}) data
 				if t == "stacked" {
 					datadogToplistStyle.SetDisplay(datadogV1.ToplistWidgetDisplay{
 						ToplistWidgetStacked: datadogV1.NewToplistWidgetStacked(
-							datadogV1.TOPLISTWIDGETLEGEND_AUTOMATIC,
```

Comment:
Since the `legend` doesn't have a default value, the request will lose `"legend":"automatic"`? Is it safe to remove it, or should we `.SetLegend(datadogV1.TOPLISTWIDGETLEGEND_AUTOMATIC)`? Does the backend default to automatic or none? cc. @brtu

Reply:
Reverted this change and added back the old defaults